apps/api/src/.example.env

```
@@ -84,3 +84,4 @@ CLERK_LONG_LIVED_TOKEN=
 
 TUNNEL_BASE_ADDRESS=
 PLAIN_SUPPORT_KEY='PLAIN_SUPPORT_KEY'
+NOVU_SECRET_KEY=
```

Typo, incorrect env var key

```suggestion
NOVU_INTERNAL_SECRET_KEY =
```